app/javascript/controllers/icon_toggle_controller.js

```diff
@@ -0,0 +1,14 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  static targets = ['icon']
+  static values = {
```

Comment:
Using values makes it possible to change with any number of icons and different icons. Makes it a little more usable.